crash-handler-process/process-manager.cpp

```diff
@@ -112,17 +114,33 @@ void ProcessManager::monitor_fnc()
 	bool criticalCrash = false;
 	bool unresponsiveMarked = false;
 	uint32_t last_responsive_check = 0;
+	std::chrono::time_point<std::chrono::system_clock> start = std::chrono::system_clock::now();
+	bool validatedProcesses = false;
 
 	while (!this->monitor->wait_or_stop()) {
 		bool detectedUnresponsive = false;
 		if (this->mtx.try_lock()) {
 			if (++last_responsive_check % 100 == 0)
 				last_responsive_check = 0;
 
+			bool willValidateProcesses = false;
```

Comment:
I try to avoid multiple time check. I do it once for the loop and only for the first several seconds of running. I assume that all process should be started by that time. So we know if the handles are valid. So actual verification is done only once. Please tell if you still do not like it. I will change it.